pom.xml

```diff
@@ -305,6 +305,7 @@
         <configuration>
           <ignoredDifferencesFile>clirr-ignored-differences.xml</ignoredDifferencesFile>
           <logResults>true</logResults>
+          <failOnError>false</failOnError>
```

Comment:
I think we want to keep having it fail on non-ignored breaking changes.

Reply:
This is pre-1.0